README.md

```diff
@@ -105,7 +138,10 @@ d4p.daalfini()
 
 daal4py patching affects performance of specific Scikit-learn functionality listed below. In cases when unsupported parameters are used, daal4py fallbacks into stock Scikit-learn. These limitations described below. If the patching does not cover your scenarios, [submit an issue on GitHub](https://github.com/IntelPython/daal4py/issues).
 
-Scenarios that are already available in 2020.3 release:
+⚠️ We support optimizations for the last 4 versions of Scikit-learn. The latest release of daal4py-2021.1 suppors Scikit-learn 0.21, 0.22, 0.23 and 0.24.
+
+<details><summary>[Click to expand] 🔥 Scenarios that are already available in 2020.3 release </summary>
```

Comment:
Probably, need to rename this to better describe the content.

Reply:
Done